packages/components/radio/src/radio.tsx

```diff
@@ -298,34 +320,37 @@ export const Radio = forwardRef(
       ...computedProps
     } = omitThemeProps(mergedProps)
 
-    const isCheckedProp =
+    computedProps.checked ??= computedProps.isChecked
+
+    const checkedProp =
       groupValue && computedProps.value
         ? groupValue === computedProps.value
-        : computedProps.isChecked
+        : computedProps.checked
 
     const onChange =
       groupProps.onChange && computedProps.value
         ? funcAll(groupProps.onChange, computedProps.onChange)
         : computedProps.onChange
 
     const {
-      isChecked,
+      checked,
       props: rest,
       getContainerProps,
       getIconProps,
       getInputProps,
       getLabelProps,
     } = useRadio({
       ...computedProps,
-      isChecked: isCheckedProp,
+      checked: checkedProp,
```

Comment:
Please remove line.

Reply:
Is “checked: checkedProp,” also deleted?

Reply:
My mistake! Never mind!